src/main/resources/swagger/rawls.yaml

```diff
@@ -3232,6 +3231,19 @@ definitions:
         type: string
         description: The id of this workflow
 
+  WorkflowQueueStatusResponse:
+    description: Information about the status of workflows and the workflow queue
+    required:
+      - estimatedQueueTimeMS
+      - workflowCountsByStatus
+    properties:
+      estimatedQueueTimeMS:
```

Comment:
yes, I went with the algorithm of last submitted workflow wait time. I totally see your point on naming ... I went with the alternate approach of keeping this name generic, and allowing the underlying algorithm to change without affecting the response payload.